CAT/model/IRT.py

```diff
@@ -52,8 +98,11 @@ def name(self):
         return 'Item Response Theory'
 
     def init_model(self, data: Dataset):
+        betas = (0.9, 0.999)
```

Comment:
add args to set these hyper parameters instead of directly setting in the function

Reply:
Also, if it is not necessary for all IRT models, use args to control